src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -457,10 +473,14 @@ private void notifyCantPlayReplay(@Nullable Integer replayId, Throwable throwabl
         .thenCompose(featuredModBean -> updateGameIfNecessary(featuredModBean, null, modVersions, simModUids))
```

Comment:
Could a check to see if the game is running also be added to the updateGameIfNecessary function? This way it doesn't even try to download if another game is running saving the network cycles as I think currently it only errors out after download when it tries to move the files